### DISCUSSION TITLE
Index important to keep?

### DISCUSSION THREAD

**Discussion by bloodconfetti** [TumblThreeApp/TumblThree#426] [Q&A (Question & Answer)]
Hello, after you've finished downloading a tumblog is it necessary to keep the Index files? is the information contained in those files useful for archival purposes, or is it only used to identify a blog prior to download?

Thanks! (Also are there any 'temporary' files one should be cleaning out to save space, etc?

**Comment by thomas694** [comment]
Hello,
the index files are only necessary for updating the blog.
There are no temporary files with the default settings.
